.github/workflows/main.yml

```
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+       node-version: '14.x'
+
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+
+    - name: Install JupyterLab
+      run: |
+        python -m pip install jupyterlab
+
+    - name: Install the extension
+      run: |
+        jlpm && jlpm run build
+
+        jupyter labextension link packages/fasta-extension --no-build
+        jupyter labextension link packages/geojson-extension --no-build
+        jupyter labextension link packages/katex-extension --no-build
+        jupyter labextension link packages/vega2-extension --no-build
+        jupyter labextension link packages/vega3-extension --no-build
+        jupyter labextension link packages/mathjax3-extension --no-build
```

These days, `install` does `link` if it is a local directory
```suggestion
        jupyter labextension install ./packages/fasta-extension --no-build
        jupyter labextension install ./packages/geojson-extension --no-build
        jupyter labextension install ./packages/katex-extension --no-build
        jupyter labextension install ./packages/vega2-extension --no-build
        jupyter labextension install ./packages/vega3-extension --no-build
        jupyter labextension install ./packages/mathjax3-extension --no-build
```